src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

```diff
@@ -549,6 +549,21 @@ public sealed partial class SqlParameter : System.Data.Common.DbParameter, Syste
         public SqlParameter(string parameterName, System.Data.SqlDbType dbType, int size) { }
         public SqlParameter(string parameterName, System.Data.SqlDbType dbType, int size, string sourceColumn) { }
         public SqlParameter(string parameterName, object value) { }
```

Comment:
And also include the explicit namespace references, for consistency. Example: System.Data.SqlDbType